ngraph/core/src/op/non_max_suppression.cpp

```diff
@@ -641,36 +624,91 @@ op::v5::NonMaxSuppression::NonMaxSuppression(
+                                                           new_args.at(2),
+                                                           m_box_encoding,
+                                                           m_sort_result_descending,
+                                                           m_output_type);
+        break;
+    case 4:
+        return std::make_shared<op::v5::NonMaxSuppression>(new_args.at(0),
+                                                           new_args.at(1),
+                                                           new_args.at(2),
+                                                           new_args.at(3),
+                                                           m_box_encoding,
+                                                           m_sort_result_descending,
+                                                           m_output_type);
+        break;
+    case 5:
+        return std::make_shared<op::v5::NonMaxSuppression>(new_args.at(0),
+                                                           new_args.at(1),
+                                                           new_args.at(2),
+                                                           new_args.at(3),
+                                                           new_args.at(4),
+                                                           m_box_encoding,
+                                                           m_sort_result_descending,
+                                                           m_output_type);
+        break;
+    default:
+        return std::make_shared<op::v5::NonMaxSuppression>(new_args.at(0),
+                                                           new_args.at(1),
+                                                           new_args.at(2),
+                                                           new_args.at(3),
+                                                           new_args.at(4),
+                                                           new_args.at(5),
+                                                           m_box_encoding,
+                                                           m_sort_result_descending,
+                                                           m_output_type);
+        break;
+    }
+}
 
-    return std::make_shared<op::v5::NonMaxSuppression>(new_args.at(0),
-                                                       new_args.at(1),
-                                                       arg2,
-                                                       arg3,
-                                                       arg4,
-                                                       arg5,
-                                                       m_box_encoding,
-                                                       m_sort_result_descending,
-                                                       m_output_type);
+using V5BoxEncoding = op::v5::NonMaxSuppression::BoxEncodingType;
```

Comment:
I didn't find any places when we use this `using V5BoxEncoding `

Reply:
Deleted.